apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -783,6 +783,30 @@ expression: "&schema"
       },
       "type": "object"
     },
+    "CacheControlSelector": {
+      "oneOf": [
+        {
+          "const": "public",
+          "description": "Boolean if it's public or not",
+          "type": "string"
+        },
+        {
+          "const": "private",
+          "description": "Boolean if it's private or not",
+          "type": "string"
+        },
```

Comment:
Can it be public and private at the same time? If not I think remove one of them.

Reply:
What about having `scope` which would return either `public` or `private` instead ?

Reply:
+1 That sounds better!